spec/helpers/hyrax/ability_helper_spec.rb

```diff
@@ -6,7 +6,7 @@
       Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC =>
         "<span class=\"label label-success\">Public</span>",
       Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_AUTHENTICATED =>
-        "<span class=\"label label-info\">%s</span>",
+        "<span class=\"label label-info\">%<name>s</span>",
```

Comment:
What is going on here?

Reply:
See: http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/FormatStringToken

Reply:
Okay, looks good.